lisa/tests/scheduler/util_tracking.py

```diff
@@ -227,6 +228,10 @@ def test_means(self) -> ResultBundle:
             apply_phase_window = functools.partial(df_refit_index, window=(phase.start, phase.end))
 
             ue_phase_df = apply_phase_window(ue_df)
+            if ue_phase_df.shape[0] <= 1:
+                self.logger.warning(f"Not enough events recorded for phase {phase.id} to proceed")
```

Comment:
I'm not sure this belongs to this PR but the only reason I can see for having an empty dataframe here is the usual cpufreq Juno R0 issue that leads to util=1023 during the whole test and therefore no util_est values at all.